lib/schema/schema.ts

```diff
@@ -110,7 +120,7 @@ export class Schema {
    */
   async generateId(): Promise<string> {
     const ulidStrategy = () => ulid()
-    return await (this.#options?.idStrategy ?? ulidStrategy)()
+    return (this.#options?.idStrategy ?? ulidStrategy)();
```

Comment:
I follow the practice of usually awaiting before returning for cleaner stack traces, but I can remove them.

Reply:
I do the same. In this particular case, the await is needed because the user might provide an async function for the idStrategy. I got the request for this from a fair number of users who wanted to do things like query a database to generate their next ID.